### DISCUSSION TITLE
how to have multiple pages

### DISCUSSION THREAD

**Discussion by Abolfazl2049** [alex8088/electron-vite#522] [Q&A]
hey im learninig electron with vue and vite (this project) .
how can i have multiple pages in my electron vite app ?
(there was a some documentations about in official website but it didnt work for me)
i would appreciate  for your answer

**Comment by alex8088** [comment]
https://electron-vite.org/guide/dev#multiple-windows-app
https://electron-vite.org/guide/hmr
